README.md

```diff
@@ -1,38 +1,38 @@
-[![Skeleton](https://user-images.githubusercontent.com/1509726/199282306-7454adcb-b765-4618-8438-67655a7dee47.png)](https://www.skeleton.dev/)
+[![Shmeleton](https://user-images.githubusercontent.com/1509726/199282306-7454adcb-b765-4618-8438-67655a7dee47.png)](https://www.Shmeleton.dev/)
 
-[![npm version](https://img.shields.io/npm/v/@skeletonlabs/skeleton?logo=npm&color=cb3837)](https://www.npmjs.com/package/@skeletonlabs/skeleton)
+[![npm version](https://img.shields.io/npm/v/@Shmeletonlabs/Shmeleton?logo=npm&color=cb3837)](https://www.npmjs.com/package/@Shmeletonlabs/Shmeleton)
```

Comment:
this url has a "j" not good. do replace please.

Reply:
normal comment but is it?

Reply:
Hello world

Reply:
review number 5, too.